web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -370,7 +370,7 @@ const TeamReviews = ({ onBack, periodId }) => {
 
   const handleLaunchDateChange = (val, period) => {
     const newDate = val?.$d;
-    const isoDate = newDate.toISOString() ?? null;
+    const isoDate = newDate?.toISOString() ?? null;
```

Comment:
When this component is initially loaded and the review period has no launch date yet, the `DatePicker` for launch date automatically opens to the calendar view and `$d` has no value. Adding the `?` here addresses that.